### PR TITLE
Prepare for release 1.0.0

### PR DESCRIPTION
**Pull request by saket**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+  - [`contourWidthWrapContent()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L328) and [`contourHeightWrapContent()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L357)
+  - [`matchXTo()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L564) and [`matchYTo()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L592)
+  - [`matchParentX()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L578) and [`matchParentY()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L606)
+  
+### Breaking changes
```
Comment (jrodbx) [inline CHANGELOG.md:12]: Not a breaking change pre-1.0?

**Review by jrodbx — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (saket) [reply CHANGELOG.md:12]: No these were introduced after `0.1.7`: https://github.com/cashapp/contour/compare/0.1.7...master

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
 # Change Log
 
-## Unreleased
+## 1.0.0 - _2020-06-11_
```
Comment (oldergod) [inline CHANGELOG.md:3]:
```suggestion
## 1.0.0 - _2020-09-21_
```

**Review by oldergod — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jrodbx) [reply CHANGELOG.md:12]: Correct, but any change introduced before 1.0 is not a breaking change.  So should we remove this header?

Comment (JakeWharton) [reply CHANGELOG.md:12]: It's breaking for users of the previous version. You'd call it out for a 0.8 release. 1.0 is the same as 0.8 here except that it comes with a future promise of compatibility and not just an indication of backwards incompatibility.

Comment (jrodbx) [reply CHANGELOG.md:12]: Got it.  So it's ok to break, but the idea is to still call it out?  👍

Comment (JakeWharton) [reply CHANGELOG.md:12]: It may be worth it to put "from 0.1" in the title to be abundantly clear about what it refers to.

Comment (saket) [reply CHANGELOG.md:12]: Sure, renamed `### Breaking changes` to `### Breaking changes (for 0.1.7)`

File: CHANGELOG.md (hunk 2 above)
Comment (saket) [reply CHANGELOG.md:3]: whoops thank you

**Review by jrodbx — APPROVED** [review]

**Pull request merged**